opendcs-rest-api-jetty/build.gradle

```diff
@@ -58,16 +58,94 @@ task createStartScript() {
         new File(binDir, "start.sh").text = """
 #!/bin/bash
 cd `dirname \$0`/..
-java \$JAVA_OPTS -jar libs/${project.name}-${project.version}.jar \$JAVA_ARGS
+java  -Djava.util.logging.config.file=logging.properties \$JAVA_OPTS -jar libs/${project.name}-${project.version}.jar \$JAVA_ARGS
             """
     }
 }
 
+task createLoggingPropertiesFile() {
```

Comment:
I think this one shouldn't get created in gradle, it should be a resource file that gets copied over